.github/workflows/test.yml

```diff
@@ -68,6 +68,7 @@ jobs:
       fail-fast: false
       matrix:
           os: [macOS-latest, windows-latest]
+          include: [eloquent, foxy]
```

Comment:
This isn't how the `include` syntax works. `include` is "extra variables for specific matrix combinations" https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations

If you're trying to do 2x2 matrix (mac+eloquent, mac+foxy, win+eloquent, win+foxy) it would be

```
matrix:
  os: [macOS-latest, windows-latest]
  ros_distribution: [eloquent, foxy]

...

target-ros2-distro: ${{ matrix.ros_distribution }}
```

If you were trying to specify (mac+eloquent, win+foxy), it would look like

```
matrix:
  os:   os: [macOS-latest, windows-latest]
  include:
    - os: macOS-latest
      ros_distribution: eloquent
    - os: windows-latest
      ros_distribution: foxy
  
```